corehq/motech/fhir/views.py

```diff
@@ -44,8 +44,7 @@ def page_url(self):
 
 
 @require_GET
-@login_or_api_key
-@require_superuser
+@api_key_auth
```

Comment:
do you think this needs the `require_edit_data` permission and the api privilege like we do for the case list API? 

https://github.com/dimagi/commcare-hq/blob/1c4b1653c7d8d2bd23ce76b61033d3a0d1ea5195/corehq/apps/hqcase/views.py#L86-L89

Reply:
Yes, I'd like to use the same permissions as the Case API (0.6) (and when the FHIR API supports modifying data, "require_**edit**_data" will feel like its name is justified).

Reply:
`require_edit_data` and `API_ACCESS` added. (fbfd515baf91564027bd9ebf95de925926894fa9 and 640e5204830e737f895270e4ef5fd3e482ea2d48)